distribution/scripts/ei/microei-start.sh

```diff
@@ -30,10 +31,11 @@ function usage() {
     echo "-v: WSO2 Enterprise Integrator version."
     echo "-m: The heap memory size of Micro Integrator. Default: $default_heap_size."
     echo "-h: Display this help and exit."
+    echo "-a: ei for EI and mei for Micro EI"
```

Comment:
Keep this as a flag for now. We only have two options.

Reply:
Corrected.